packages/migrate/src/commands/MigrateDiff.ts

```diff
@@ -1,6 +1,6 @@
 import path from 'path'
 import type { Command } from '@prisma/sdk'
-import { arg, format, HelpError, isError, getSchemaPath, link, getCommandWithExecutor } from '@prisma/sdk'
+import { arg, format, HelpError, isError, link } from '@prisma/sdk'
 import chalk from 'chalk'
```

Comment:
Does the absence of loadEnvAndPrint here mean that we don't load the .env at all? It could be a problem with `--from-schema-datasource`, because there users would expect the env vars from .env to be accessible in the schema.

Reply:
I think so, shows a missing test, I'll check that now